scapy/fields.py

```diff
@@ -630,6 +630,22 @@ def i2repr(self, pkt, x):
         return ByteField.i2repr(self, pkt, x)
 
 
+class LEX3BytesField(XByteField):
```

Comment:
One last thing:
- it would be better to have it named LEXThree... (3->Three) for consistency
- We (by convention) make the X fields import the non-X fields. It is the opposite here

It would also be nice to have some regressions tests (add them to test/fields.uts). Have a look on how there are built.

Sorry to be picky :/

Reply:
@gpotter2: refactoring `X3BytesField` to `XThreeBytesField` will touch 83 lines in 20 files - do you want me to do that? 

I've reversed the inheritance, will look at the testing.